### ISSUE TITLE
Error with Image loading node AttributeError: 'Tensor' object has no attribute 'convert'

### ISSUE THREAD

**Issue by BARETFabrice** [WASasquatch/was-node-suite-comfyui#38]
Error occurs when trying to load an image, ComfyUI repository is up to date

```
got prompt
Traceback (most recent call last):
  File "C:\Users\BARET\Desktop\App\ComfyUI\execution.py", line 182, in execute
    executed += recursive_execute(self.server, prompt, self.outputs, x, extra_data)
  File "C:\Users\BARET\Desktop\App\ComfyUI\execution.py", line 58, in recursive_execute
    executed += recursive_execute(server, prompt, outputs, input_unique_id, extra_data)
  File "C:\Users\BARET\Desktop\App\ComfyUI\execution.py", line 58, in recursive_execute
    executed += recursive_execute(server, prompt, outputs, input_unique_id, extra_data)
  File "C:\Users\BARET\Desktop\App\ComfyUI\execution.py", line 67, in recursive_execute
    outputs[unique_id] = getattr(obj, obj.FUNCTION)(**input_data_all)
  File "C:\Users\BARET\Desktop\App\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 3220, in load_image
    return (image.convert('RGB'), mask)
AttributeError: 'Tensor' object has no attribute 'convert'
```

I send "C:\Users\BARET\Desktop\App\ComfyUI\output\wd_156_m_cup\img7.jpg" as path

**Comment by WASasquatch** [comment]
Should be patched in 2ff0c04a656d1d69f8eb24d76159dfd8cbf59a04

**Comment by BARETFabrice** [comment]
Yes it works
